### PR TITLE
Add tests for st2-pack-install CLI tool

### PR DESCRIPTION
**Pull request by Kami**
This pull request adds tests for the new `st2-pack-install` CLI tool added in https://github.com/StackStorm/st2/pull/4256.

### CODE REVIEW

File: packs/tests/actions/chains/test_pack_install_tool_tests.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+vars:
+    base_repo_url: "https://github.com/StackStorm"
+    # Note: Pack 1 should have no external dependencies beyond Python stdlib ones.
+    pack_to_install_1: "csv"
```
Comment (armab) [inline packs/tests/actions/chains/test_pack_install_tool_tests.yaml]: Would be nice to add a version selector here (like `csv=0.4.2`) which will verify more complex examples alltogether, - something that was an issue in st2 PR that implemented this script.

Comment (Kami) [reply packs/tests/actions/chains/test_pack_install_tool_tests.yaml]: Agreed, added in b6f66ecb30e3b473a75df3869bff911d7b458935.

Comment (Kami) [reply packs/tests/actions/chains/test_pack_install_tool_tests.yaml]: (More tests for various edge cases also wouldn't hurt of course, hopefully I can come back to that in the near future)
